### PR TITLE
Add a test for new `Hsteps` keyword argument

### PR DESCRIPTION
**Pull request by newton-per-sqm**
Implementing a necessary new test for PR https://github.com/ubermag/oommfc/pull/138 (Adding Hsteps as new keyword argument to HysteresisDriver for stepped hysteresis loops).

Test structure of new `TestHysteresisDriver` class is now (again) similar to already existing `TestMinDriver` class.

Just a draft idea, if you want to, we could also just write a second function. IMHO the class structure is nicer.

### CODE REVIEW

**Comment by lang-m** [comment]
@newton-per-sqm I have reverted the structure to simple functions (instead of the class) and added additional fixtures to avoid duplication.

**Review by samjrholt — APPROVED** [review]

**Comment by lang-m** [comment]
We should wait with merging until https://github.com/ubermag/oommfc/pull/151 is merged.

**Comment by lang-m** [comment]
@lang-m Resolve conflicts after merging #51

File: micromagnetictests/tests/test_get_tests.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 def test_get_tests():
     tests = list(mt.get_tests())
-    assert len(tests) == 33
+    assert len(tests) == 31
```
Comment (samjrholt) [inline micromagnetictests/tests/test_get_tests.py:6]:
```suggestion
    assert len(tests) == 34
```

**Comment by samjrholt** [comment]
> @lang-m Resolve conflicts after merging #51

I have merged #51 and resolved conflicts. Please merge and add to the change log if you are happy.

**Pull request merged**